packages/typespec-ts/src/utils/emitUtil.ts

```diff
@@ -61,7 +61,7 @@ async function emitFile(
   const microsoftHeader = isAzureFlavor
     ? `// Copyright (c) Microsoft Corporation.\n`
     : "";
-  const licenseHeader = `${microsoftHeader}// Licensed under the MIT license.\n`;
+  const licenseHeader = `${microsoftHeader}// Licensed under the MIT License.\n`;
```

Comment:
codegen was using lower case so I am not 100% sure why some test baselines have upper case and some have lower case.

Reply:
Could you point out to me where's the upper case baselines test?

Reply:
@qiaozha maybe those are outdated? https://github.com/Azure/autorest.typescript/blob/669662f259c957f71474e4e2cd611f9a200dc749/packages/autorest.typescript/test/smoke/generated/storage-resource-manager/samples-dev/operationsListSample.ts#L10

Reply:
https://github.com/Azure/autorest.typescript/blob/669662f259c957f71474e4e2cd611f9a200dc749/packages/autorest.typescript/src/generators/samples/hlcSamples.ts.hbs#L3 is this the reason?

Reply:
for sample generation, we use handle bars template.

Reply:
I think the operation interface is because of this https://github.com/Azure/autorest.typescript/blob/669662f259c957f71474e4e2cd611f9a200dc749/packages/autorest.typescript/src/typescriptGenerator.ts#L120